nulog-inference-service/start-nulog-inference.py

```diff
@@ -14,8 +14,9 @@
 import pandas as pd
 from elasticsearch import AsyncElasticsearch
 from elasticsearch.helpers import async_bulk
+from HyperParamaters import HyperParameters
 from nats.aio.errors import ErrTimeout
-from NulogServer import NulogServer
+from NulogServer import MIN_LOG_TOKENS, NulogServer
```

Comment:
is this import `MIN_LOG_TOKENS` correct?

Reply:
Nope, that's a typo, will fix.